Mathlib/CategoryTheory/Adjunction/Basic.lean

```diff
@@ -63,11 +63,9 @@ structure Adjunction (F : C ⥤ D) (G : D ⥤ C) where
   unit : 𝟭 C ⟶ F.comp G
   /-- The counit of an adjunction -/
   counit : G.comp F ⟶ 𝟭 D
-  -- Porting note: It's strange that this `Prop` is being flagged by the `docBlame` linter
-  /-- Naturality of the unit of an adjunction -/
+  /-- The relationship between the unit and hom set equivalence of an adjunction -/
```

Comment:
It's not anymore, but having a docstring doesn't hurt. Even better if the docstring is correct :-)